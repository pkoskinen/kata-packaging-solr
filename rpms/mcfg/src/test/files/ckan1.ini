hostname = %%HOSTNAME%%
foo = bar
email = %%ADMIN_EMAIL%%