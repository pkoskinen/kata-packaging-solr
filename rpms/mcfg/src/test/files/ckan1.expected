hostname = kata-test1.csc.fi
foo = bar
email = kata-admin@csc.fi